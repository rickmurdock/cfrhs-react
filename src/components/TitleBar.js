import React from 'react';
import { Link } from 'react-router-dom';

const TitleBar = () => {
  return (
    <div className="container-fluid">
      <div className="row align-items-center">
        <div className="col-sm-12 col-md-4 col-lg-3">
          <div className="container p-0 text-center">
            <Link to="/">
              <img className="logo img-fluid" src={require('../images/CFRHSLogoSmall.JPG')} width="350" alt="Central Florida Railway Historical Society Logo"/>
            </Link>
          </div>
        </div>
        <div className="col-sm-12 col-md-8 col-lg-6">
          <h1 className="text-center title mx-auto">Central Florida Railway Historical Society, Inc.</h1>
        </div>
        {/*<div className="col-md-3">
            <h2 style={{color:"orange", transform:"rotate(6deg)", textShadow:" 4px 3px 4px #000000"}}><b> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;UNDER CONSTRUCTION</b></h2>
        </div>*/}
      </div>
    </div>
  );
};

export default TitleBar;