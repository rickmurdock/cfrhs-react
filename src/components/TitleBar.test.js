import React from 'react';
import renderer from 'react-test-renderer';
import TitleBar from './TitleBar';

test('TitleBar renders correctly', () => {
    const tree = renderer.create(
        <TitleBar />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
