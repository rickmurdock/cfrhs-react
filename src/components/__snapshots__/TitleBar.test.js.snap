// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleBar renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row align-items-center"
  >
    <div
      className="col-sm-12 col-md-4 col-lg-3"
    >
      <div
        className="container p-0 text-center"
      >
        <img
          alt="Central Florida Railway Historical Society Logo"
          className="logo img-fluid"
          src="CFRHSLogoSmall.JPG"
          width="350"
        />
      </div>
    </div>
    <div
      className="col-sm-12 col-md-8 col-lg-6"
    >
      <h1
        className="text-center title mx-auto"
      >
        Central Florida Railway Historical Society, Inc.
      </h1>
    </div>
  </div>
</div>
`;
